### PR TITLE
fix: PANIC by global cc error object in datacollection issue #4313

### PR DESCRIPTION
**Pull request by mrmgxxxx**
### 修复的问题：
- 修复datacollection模块因全局CCError对象未能正确初始化产生panic的问题

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=4314) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=4314) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-cmdb?pullRequest=4314) it.</sub>

File: src/scene_server/datacollection/app/server.go [hunk 1]
```diff
@@ -59,6 +60,9 @@ func Run(ctx context.Context, cancel context.CancelFunc, op *options.ServerOptio
 		return fmt.Errorf("new backbone failed, err: %v", err)
 	}
 
+	// set global cc errors.
+	errors.SetGlobalCCError(engine.CCErr)
```
Comment (fireyun) [inline src/scene_server/datacollection/app/server.go:64]: hostserver也有类似的问题，用到了auditlog，但没初始化全局错误对象，建议把这个初始化操作放在NewBackbone方法里，所有模块都会调用，防止以后其他模块用到时忘记初始化。

Comment (mrmgxxxx) [reply src/scene_server/datacollection/app/server.go:64]: hostserver 是现存必现的吗

Comment (mrmgxxxx) [reply src/scene_server/datacollection/app/server.go:64]: 这个讨论过，暂行修复datacollection有问题的模块，后面做统一规整

**Pull request merged**